Add tests

Wikitelmex opened the pull request:
- add calculate.js tests to receive and compare the object that it generates
- add operate.js tests to checkout calculations made between numbers.ToString
- add Calculator.js test, to check if this exists into screen and if it fits to the snapshot
- add App test, to check if App exists in the vDOM tree
- add CalculatorPage.js snapshot test
- add HomePage.js snapshot test
- add QuotePage.js snapshot test
- add APP main component snapshot test
- fix linters

wikitelmex1 committed:
Add tests

- add calculate.js tests to receive and compare the object that it generates
- add operate.js tests to checkout calculations made between numbers.ToString
- add Calculator.js test, to check if this exists into screen and if it fits to the snapshot
- add App test, to check if App exists in the vDOM tree

GzimAsani commented on src/components/Calculator/__snapshots__/Calculator.test.js.snap:
```diff
@@ -0,0 +1,154 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```
- [x] There should be the usage of Jest snapshots for testing all the components. Currently I can see snapshots only for Calculator

GzimAsani reviewed (changes requested):
# Hi @Wikitelmex                             
Good job so far! Your code looks fine, good job :+1:  :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight:
- Good job with the gitflow :+1:
- Good PR description :clap: 

## Required Changes ♻️ 
- [x] _Check inline comments for more information :arrow_double_down:_ 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

wikitelmex1 committed:
Required Changes first submit

- add CalculatorPage.js snapshot test
- add HomePage.js snapshot test
- add QuotePage.js snapshot test

wikitelmex1 committed:
aditional snapshot test

- add APP main component snapshot test
-fix linters

GzimAsani reviewed (approved):
# STATUS: APPROVED :100:         
## Thank you for implementing all the project requirements, you've done a great job :trophy: There is nothing else to say other than... it's time to merge it :champagne:  :tada: Congratulations

Cheers and Happy coding! :clap:  :computer:  :wine_glass: 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).
